feature-libs/product-configurator/common/components/service/configurator-router-extractor.service.spec.ts

```diff
@@ -181,6 +181,18 @@ describe('ConfigRouterExtractorService', () => {
         })
         .unsubscribe();
     });
+
+    it('should tell from the URL if we need to fetch a configuration for an expert mode', () => {
+      mockRouterState.state.queryParams = { expMode: 'true' };
+      let routerData: ConfiguratorRouter.Data;
+      serviceUnderTest
+        .extractRouterData()
+        .subscribe((data) => {
+          routerData = data;
+          expect(routerData.expMode).toBe(true);
+        })
+        .unsubscribe();
+    });
   });
 
```

Comment:
can you plz add at least one test for the default behaviour, so 
when queryParam expMode is not present
then expect(routerData.expMode).toBe(false);

Reply:
@Larisa-Staroverova: did you saw this?

Reply:
Done!
PS: this comment is not shown in this [view](https://github.com/SAP/spartacus/pull/16034/files)